src/main/java/teammates/common/util/TimeHelper.java

```diff
@@ -251,6 +251,14 @@ public static String formatDateTimeForInstructorHomePage(Date date) {
         return sdf.format(date);
     }
 
+    public static String formatDateTimeForFeedbackSubmissionEditPage(Date date, double timeZone) {
+        StringBuffer formattedDateTime = new StringBuffer(formatTime12H(date));
+        formattedDateTime.append(" (")
+                         .append(getTimeZoneFromDoubleOffset(timeZone).getID().replace("GMT", "UTC "))
```

Comment:
Other than the brackets around the timezone info and the colon separator between hours and minutes in the time zone, no major difference at all.

Output from currently committed changes: `Wed, 12 Jul 2017, 10:00 AM (UTC +05:30)`
Output from `formateDateTimeForComments` : `Wed, 12 Jul 2017, 10:00 AM UTC+0530`

Would it be okay to just use `formateDateTimeForComments` entirely? I don't think those differences are too significant.